daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -551,6 +554,38 @@ codegenTests codegenDir damlTypes = testGroup "daml codegen" (
                         contents <- listDirectory outDir
                         assertBool "bindings were written" (not $ null contents)
 
+createDamlAppTests :: TestTree
+createDamlAppTests = testGroup "create-daml-app" [gettingStartedGuideTest | not isWindows]
+  where
+    gettingStartedGuideTest = testCase "Getting Started Guide" $
+      withTempDir $ \tmpDir -> do
+        let tsLibs = ["daml-types", "daml-ledger", "daml-react"]
+        forM_  tsLibs $ \tsLib -> do
+          srcDir <- locateRunfiles $ mainWorkspace </> "language-support" </> "ts" </> tsLib </> "npm_package"
+          copyDirectory srcDir (tmpDir </> tsLib)
+        BSL.writeFile (tmpDir </> "package.json") $ Aeson.encode $ object
+          [ "private" .= True
+          , "workspaces" .= ["create-daml-app/daml-ts", "create-daml-app/ui" :: String]
+          , "resolutions" .= object
+              [ pkgName .= ("file:" ++ tsLib)
+              | tsLib <- tsLibs, let pkgName = "@" <> T.replace "-" "/"  (T.pack tsLib)
+              ]
+          ]
+        withCurrentDirectory tmpDir $ do
+          callCommandSilent "daml new create-daml-app create-daml-app"
+        let cdaDir = tmpDir </> "create-daml-app"
+        withCurrentDirectory cdaDir $ do
+          callCommandSilent "daml build"
+          callCommandSilent "daml codegen ts -o daml-ts .daml/dist/create-daml-app-0.1.0.dar"
+        doesFileExist (cdaDir </> "ui" </> "build" </> "index.html") >>=
```

Comment:
We should add `assertFileExists` to `DA.Test.Util` at some point.

Reply:
I'll take care of it.